config/monitoring/metrics/prometheus/100-prometheus-scrape-config.yaml

```diff
@@ -26,6 +26,33 @@ data:
       scrape_timeout: 10s
       evaluation_interval: 30s
     scrape_configs:
+    # Controller endpoint
```

Comment:
I assume the endpoint `/metrics` for controller and webhook have been added. This RP is just to config Prometheus to scrap. Is my understanding correct?

Reply:
For controller it is added. Not for webhook yet. The single line change in controller/main.go does all the magic thanks to @akyyy

Reply:
Now I got it. The exporter decides where to send the metrics. Reporter are used to report metrics. I was thinking this PR was setting up both reporters and exporter. Have we set up reporters for controller and webhook to report metrics?

Reply:
We need to set up webhooks once we have metrics for webhooks. But for controller, this scrape config and the change above is all we need.